src/scss/components/layout/_index.scss

```diff
@@ -136,7 +135,7 @@ body.hxHorizontal {
   /* Legacy Grid Implementation */
```

Comment:
Please update single line comment to use double slash comment `//`:

```scss
// Legacy Grid Implementation
```

Reply:
Please check the changes in latest commit